ocp_resources/utils/resource_constants.py

```diff
@@ -27,6 +27,7 @@ class Condition:
         RECONCILE_COMPLETE: str = "ReconcileComplete"
         READY: str = "Ready"
         FAILING: str = "Failing"
+        NETWORK_READY = "NetworkReady"
```

Comment:
This is specific to the `UserDefinedNetwork` resource.
Are you sure it needs to be place here?

Reply:
We decided that all statuses and conditions will be here.